requirements.txt

```diff
@@ -0,0 +1,163 @@
+aiohttp==3.8.5
```

Comment:
was this added by accident?

Reply:
The file or the dependency?

The file: no, this is meant to make it easier to get set up for development.  That said we could add it in a different PR if it's too much scope creep for this one.

The dependency: unlikely, it wasn't added directly but is probably a peer dependency of one of the other libraries we are using.

Reply:
I meant the file, usually I install dependencies with `pip install -e .` and they get loaded from `setup.py`

Add `requirements.txt` is fine, as long as we keep it up to date with `setup.py`